src/community/propose-a-content-change-using-github/index.md.njk

```diff
@@ -0,0 +1,93 @@
+
+## 1. Go to the page you want to edit
+
+At the bottom of every page in the Design System you will find a section called "Help improve this page". 
+
+Follow the link to propose a change to the page. This will take you to the page's markdown file. 
+
+You might be told you need to fork the repository to make changes. This is nothing to worry about, it just means making a copy of the Design System that you can edit. Select "fork this repository and propose changes" to continue. 
+
+![](fork-this-repo.png)
+
+## 2. Edit the page file
+
+Edit the markdown to make your change. 
+
+Here is an example showing how to update the description of the checkboxes component.
+
+![The checkboxes description in the file in GitHub with an uppercase C in the word checkboxes](checkboxes-uppercase.png)
+
+In this example, the uppercase ‘C’ on the word ‘Checkbox’ has been changed to lowercase. 
+
+![The checkboxes description in the file in GitHub after the C in the word checkboxes has been changed to lowercase](checkboxes-lowercase.png)
+
+## 3. Propose a change
+
+Once you’re happy, scroll to the bottom of the page until you reach the button that says ‘propose file change’.
+
+Add a short description of what you have changed in the first field. This information will be added to the file’s changelog. Try to be as clear as possible, so that future users can understand what you have done. 
+
+If you need to provide information about why you made the change, you can add more detail in the larger field below.
+
+![Propose file change view in GitHub. There is a summary of the change and a longer description beneath it. There are 2 buttons, one with the option to propose the change and one to cancel it.](propose-file-change.png)
+
+When you are happy with your description, select ‘propose file change’. You will have a chance to review and confirm your changes on the next page.
+
+## 4. Confirm your changes
+
+You will be shown a confirmation page where you can review the changes you’ve made. 
+
+If you spot a mistake, you can go back to the previous page and correct it. 
+
+If you are happy with your changes, select ‘Create pull request’. You will have one more chance to review your change on the next page before you submit it to the GOV.UK Design System team to review.
+
+![Create pull request view in GitHub showing a before and after view of the file you have changed and a button labelled create pull request.](create-pull-request-first-view-github.png)
+
+## 5. Create a pull request
+
+A pull request is a request sent to the team who maintains the project you’re making changes to—in this case, the GOV.UK Design System team. 
+
+It tells them what you’ve changed and gives them the option to ‘pull’ or add your changes into the project, where it can then be published. 
+
+At this point, you can write additional comments about the work you’ve done if you want to.
+
+These will be shared with the GOV.UK Design System team when you submit your proposed change. 
```

Comment:
Suggest sticking with 'pull request' now we've introduced it - it might now be confusing if we now switch between 'propose a change' and 'submit a pull request'. 

```suggestion
These comments will be shared with the GOV.UK Design System team when you submit the pull request. 
```